benchmarks/htap/lib/controller.py

```diff
@@ -126,7 +127,8 @@ def _prepare_stats_db(self):
 
     def run(self):
         begin = datetime.now()
-        latest_delivery_date = self._query_latest_delivery_date()
+        range_delivery_date = self._query_range_delivery_date()
+        min_date = range_delivery_date[0]
```

Comment:
minor: why do you keep `min_date` in an extra variable but the latest date not?

Reply:
i tried to keep the diff minimal